src/js/components/sharedComponents/stickyHeader/DownloadIconButton.jsx

```diff
@@ -34,6 +35,25 @@ const DownloadIconButton = ({ onClick, downloadInFlight }) => {
     const buttonText = downloadInFlight ? 'Preparing Download...' : 'Download';
     const icon = downloadInFlight ? 'spinner' : 'download';
 
+
+    if (alternativeStyle) {
```

Comment:
I think this would make more sense as a separate component instead of the `alternativeStyle` prop. I would find it confusing that it lives in the `stickyHeaader` directory when that's not where it's used, and is called an `IconButton` but this version has text and no icon.